.github/workflows/tests.yml

```diff
@@ -7,7 +7,27 @@ on:
     branches: [ master ]
 
 jobs:
+
+  check_code_quality:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.7"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[dev]
+      - name: Check quality
+        run: |
+          black --check --line-length 119 --target-version py36 tests trl
```

Comment:
If I'm not mistaken, you can just run `make quality` here. this has the advantage of having a single source of truth for the quality logic

Reply:
done